### DISCUSSION TITLE
Support for Output to external video I/O eg. AJA / Blackmagic

### DISCUSSION THREAD

**Discussion by raketentyp** [videovillage/community#94] [Ideas]
Heyhey!

It would be great if you could implement sending the Video Stream to an external Video I/O for color calibrated Video Hardware.

Thank you in advance!

**Comment by cayubal** [comment]
Hi Greg!

Any news/plans for this? 
As far as I remember I saw it on the feature roadmap when I first subscribed to Screen in 2019...

Would be awesome to get this implemented!

Cheers!

**Reply by gregcotten** [reply]
It's still happening, just wasn't a priority at the time. Now actively being worked on!

**Reply by cayubal** [reply]
That's great news! Thanks for the heads up!

**Comment by lotero5** [comment]
Hello!

Any update on this? It will be not just great, but indispensable for a lot of us.

**Comment by Bastizor** [comment]
Hi @gregcotten,

Is it still on the roadmap ?

**Comment by lotero5** [comment]
Hello team!

Any update on this VERY important request? I know and understand that it was not a priority at one point, but it will be great if we can get an update and a clear path on your development plans.
THANKS!!!

**Comment by Bastizor** [comment]
@gregcotten

**Comment by Bastizor** [comment]
Is the Screen app dead ?

**Reply by gregcotten** [reply]
Still figuring that out

**Reply by Bastizor** [reply]
Thank you for the follow up about this.

**Reply by cayubal** [reply]
So @gregcotten: we went from 
> "It's still happening, just wasn't a priority at the time. Now actively being worked on!"

to Screen being dead.. maybe..?

Any clarification would be much appreciated! Thnks

**Reply by Bastizor** [reply]
@gregcotten

**Reply by Bastizor** [reply]
@gregcotten

**Reply by lotero5** [reply]
How can it be dead if it is the BEST player out there!  Does it need better PR for visibility?

**Comment by mwjburton** [comment]
Can we have official response from Video Village as to what the plans are for Screen please.

Now that the subscription has been removed and its become a perpetual license, does this mean there will be no further development of the Screen software or can we confidently purchase licenses for shows knowing Video Village are actively working to address issues and add features each year? If its EOF, please say so officially.

**Reply by Bastizor** [reply]
@gregcotten

**Reply by gregcotten** [reply]
Hey, I don't really have anything official to state at the moment, but can provide some clarification.

First, please know you only need to ever purchase a single license, there is no need for purchasing additional licenses for "shows" unless you need more seats for some reason.

Screen will continue to receive updates as long as Video Village is still a company. Screen is unfortunately not a successful product, and never was one. Turning off the yearly subscription for the customers subscribed and converting them to perpetual was more of a technical challenge for us than a financial one. Disabling the subscription from a financial standpoint was **quite insignificant**, sadly. We can't even afford to pay a software engineer part time to work on Screen with the profits from its sales and subscriptions.

Filmbox and Scatter have been the saving grace of this company, and are pretty much the only reason Screen and Lattice are still available for sale, as we would have likely closed shop by now if not for the success of those products.

I'll continue to make small, incremental improvements to Screen (and Lattice), but any major features will take way more time as I'm the only engineer supporting these products. Bugfixes will of course always be a priority, and some things we even get for basically free via Apple SDK improvements (HDR HEIC screenshots in macOS Sonoma!).

**Reply by Bastizor** [reply]
Appreciate the clarification @gregcotten .

**Reply by mwjburton** [reply]
Thanks Greg, appreciate the open and honest post. Its super sad to hear its not been picked up by enough people. Its a crying shame to be honest. This tool is beyond needed. I'll keep singing its praises and thank you for continuing to support the software in any way you can.
